xml/System/ArgumentException.xml

```diff
@@ -376,8 +376,8 @@
 |Property|Value|  
 |--------------|-----------|  
 |<xref:System.Exception.InnerException%2A>|The inner exception reference.|  
-|`Message`|The error message string.|  
-|`ParamName`|The parameter name string.|  
+|<xref:System.Exception.Message%2A>|The localized error message string.|  
+|<xref:System.Exception.ParamName%2A>|The parameter name string.|  
```

Comment:
This should be \<xref:System.ArgumentException.ParamName%2A>

Reply:
Thanks @rpetrusha - good spot!

Now you've pointed it out - I've noticed the other xref's are inconsistent in the file. Some reference `<xref:System.ArgumentException.Message%2A>`, others `<xref:System.Exception.Message%2A>`. Should they all be referencing ArgumentException, or does it not matter?